app/models/debate.rb

```diff
@@ -127,14 +127,11 @@ def anonymous_votes_ratio
   end
 
   def after_commented
-    save # updates the hot_score because there is a before_save
+    save # update cache when it has a new comment
```

Comment:
I tried to delete this method, but if I do there was a test that failed. The test says "update cache when it has a new comment". That's why I changed the text of the comment in the code.